code/modules/surgery/brain_surgery.dm

```diff
@@ -0,0 +1,41 @@
+/datum/surgery/brain_surgery
+	name = "brain surgery"
+	steps = list(
+	/datum/surgery_step/incise,
+	/datum/surgery_step/retract_skin,
+	/datum/surgery_step/saw,
+	/datum/surgery_step/clamp_bleeders,
+	/datum/surgery_step/fix_brain,
+	/datum/surgery_step/close)
+
+	species = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list("head")
+	requires_bodypart_type = 0
+
+/datum/surgery_step/fix_brain
+	name = "fix brain"
+	implements = list(/obj/item/hemostat = 85, /obj/item/screwdriver = 35, /obj/item/pen = 15) //don't worry, pouring some alcohol on their open brain will get that chance to 100
+	time = 120 //long and complicated
+
+/datum/surgery/brain_surgery/can_start(mob/user, mob/living/carbon/target)
+	var/obj/item/organ/brain/B = target.getorganslot("brain")
+	if(!B)
+		to_chat(user, "It's hard to do surgery on someone's brain when they don't have one.")
+		return FALSE
```

Comment:
Don't we need a return true here?

Reply:
pretty sure that can_start shouldn't be true without a brain

Reply:
I meant directly below.  There is no path that returns true for this proc.

Reply:
So....  this proc can still NEVER return true.

Are you SURE this won't cause an issue?